src/pages/resultsView/mutation/MutationMapper.tsx

```diff
@@ -61,6 +64,88 @@ export default class MutationMapper extends React.Component<IMutationMapperProps
         };
     }
 
+    @computed get mutationRateSummary() {
+        if (this.props.store.gene.result &&
+            this.props.store.patientIds.result &&
+            this.props.store.mutationData.result &&
+            this.props.store.mskImpactGermlineConsentedPatientIds.result) {
+
+            const germlineMutationRate = this.germlineMutationRate(this.props.store.mutationData.result,
+                                                                   this.props.store.patientIds.result,
+                                                                   this.props.store.mskImpactGermlineConsentedPatientIds.result)
+            const germlineMutationRateElement:JSX.Element|null =
+                (this.props.store.mskImpactGermlineConsentedPatientIds.result.length
+                    > 0 || germlineMutationRate > 0)?
+                (
+                    <span style={{paddingRight: 5}}>
+                     Germline Mutation Rate:
+                         <span style={{paddingLeft:2}}>
+                            {germlineMutationRate.toFixed(1)}%,
+                         </span>
+                    </span>
+                )
+                : null;
+
+            return (
+                <h4 style={{paddingBottom:2,paddingTop:2}}>
+                    {this.props.store.gene.result.hugoGeneSymbol}:
+                         [{germlineMutationRateElement}
+                      Somatic Mutation Rate:
+                         <span style={{paddingLeft:2}}>
+                         {this.somaticMutationRate(this.props.store.mutationData.result,
+                                                   this.props.store.patientIds.result).toFixed(1)}%]
+                         </span>
+                </h4>
+            );
+        } else {
+            return '';
+        }
+    }
+
+   private germlineMutationRate(mutations: Mutation[],
+       patientIds: string[],
+       mskImpactGermlineConsentedPatientIds: string[]) {
+
+       const nrCasesGermlineMutation:number =
+           _.chain(this.props.store.mutationData.result)
+           .filter((m:Mutation) => (m.mutationStatus === MUTATION_STATUS_GERMLINE))
+           .map('patientId')
+           .filter((patientId:string) => {
+               // only include patients that constented to germline testing in
+               // mskimpact
+               if (mskImpactGermlineConsentedPatientIds.length > 0) {
+                   return (
+                       mskImpactGermlineConsentedPatientIds.indexOf(patientId)
+                       > -1
+                   )
+               } else {
+                   return true;
```

Comment:
This filter is actually redundant, since there would not be germline mutations if no consent was given. The idea was that we would consider all patients as having had germline mutation testing if there is no information about their consent (i.e. this study is not impact). Better to remove the filter maybe, it just obfuscates the error if the underlying data is incorrect.